about.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" 
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" 
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <title>WebTekhy | About page</title>
+</head>
+<body id="hideOverflow">
+    <header class="header" id="header">
+        <div class="logo">
+            <a href="index.html"><img class="logo-img" src="images/webtekhy-icon.png" alt="webtekhy icon"></a>
+        </div>
+        <nav id="menu_bar" class="navbars">
+            <i id="menuBarImg" class="fa fa-bars" aria-hidden="true"></i>
+            <div id="close_bar" class="close-menu">
+                <i id="close_menu" class="close" >&times;</i>  
+            </div>
+            <ul id="menu_list" class="navis">
+                <li class="nav-item"><a href="about.html" class="nav-link abt-nav">About</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Program</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Join</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">Sponsor</a></li>
+                <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                <li class="nav-item"><a href="#" class="nav-link wt-nav">WT Campaign</a></li>
+            </ul>
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure you add a link from the about page to the home page. You can add a `home` option in the navbar menu so users can navigate to the homepage.